migrations/migrations/0012_committee_user_relation.py

```diff
@@ -88,31 +160,23 @@ def handle_user_create_event(
         if not cml_committee_ids:
             return None
 
-        # change cml-fields and create event for committee
-        for committee_id in cml_committee_ids:
-            event.data.pop(f"committee_${committee_id}_management_level")
-        event.data["committee_$_management_level"] = [cml_permission]
-        event.data[f"committee_${cml_permission}_management_level"] = cml_committee_ids
         return [
-            event,
-            *[
-                ListUpdateEvent(
-                    f"committee/{committee_id}",
-                    {
-                        "add": {
-                            f"user_${cml_permission}_management_level": [user_id],
-                            "user_$_management_level": [cml_permission],
-                        }
-                    },
-                )
-                for committee_id in cml_committee_ids
-            ],
+            ListUpdateEvent(
+                f"committee/{committee_id}",
+                {
+                    "add": {
+                        f"user_${cml_permission}_management_level": [user_id],
+                        "user_$_management_level": [cml_permission],
+                    }
+                },
+            )
+            for committee_id in cml_committee_ids
         ]
 
     def handle_user_delete_event(
-        self, user_id: int, event: DeleteEvent
+        self, event: DeleteEvent, user: Dict[str, Any], user_id: int
     ) -> Optional[List[BaseEvent]]:
-        user = self.new_accessor.get_model_ignore_deleted(f"user/{user_id}")[0]
+        # user = self.new_accessor.get_model_ignore_deleted(f"user/{user_id}")[0]
```

Comment:
Mmh, don't you want to delete this line?

Reply:
done